cnf.1

```diff
@@ -29,6 +29,12 @@ Try installing with:
 bash(1), zsh(1), fish(1), libsolv(3)
 .SH BUGS
 No support for localization other than UTF-8.
+.PP
```

Comment:
Surprisingly noone ever complained about neither in this variant, neither in an original Python implementation. But good too have this fixed.